api/api.go

```diff
@@ -3,26 +3,52 @@ package api
 import (
 	"encoding/json"
 	"net/http"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/gorilla/schema"
+	"github.com/rs/zerolog/log"
 )
 
 type Error struct {
-	Code int
-	Message string
+	Code      int    `json:"code"`
+	Message   string `json:"message"`
+	SupportId string `json:"support_id"`
 }
 
 type QueryParams struct {
-	ChainID string
+	FilterParams map[string]string `schema:"-"`
```

Comment:
Can you do larger/smaller/equal here?

Reply:
yes according to the spec filter operators are bart of the filter key. e.g. `filter_block_number_gte=6666`